### PR TITLE
 Update structured-merge-diff version

### PR DESCRIPTION
**Pull request by jennybuckley**
And make necessary changes to schemaconv to accept the changes

@apelisse 

Edit: also needed to run `GO111MODULE=on go mod tidy`

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -17,6 +17,7 @@ require (
 	github.com/google/gofuzz v0.0.0-20161122191042-44d81051d367
 	github.com/googleapis/gnostic v0.0.0-20170426233943-68f4ded48ba9
 	github.com/json-iterator/go v0.0.0-20180612202835-f2b4162afba3
+	github.com/kr/pretty v0.1.0 // indirect
```
Comment (apelisse) [inline go.mod]: @liggitt is it supposed to add these?

Comment (liggitt) [reply go.mod]: running `go mod tidy` should clean it up. `go get` will speculatively add things it's unsure about, and `go mod tidy` will clean them up once it verifies all os/arch/build tag combinations don't need it

Comment (jennybuckley) [reply go.mod]: I guess not, it looks like it succeeds the tests when i remove this line

Comment (jennybuckley) [reply go.mod]: Not sure how this got here

Comment (jennybuckley) [reply go.mod]: When I ran go mod tidy, it didn't remove this, so I just removed it manually.

**Pull request merged**